bootstrap.sh

```diff
@@ -29,6 +29,6 @@ if [ ! -z "$XCS_BOT_ID"  ]; then
 fi
 
 if [ ! -f Carthage/Cartfile.resolved ]; then
-  carthage bootstrap $CARTHAGE_VERBOSE --platform ios --color auto --no-use-binaries
+  carthage bootstrap $CARTHAGE_VERBOSE --platform ios --color auto
```

Comment:
Yes, the reason is we don't compile LP, but we just import the binary, and the _--no-use-binaries_ option specifically disallows importing the binary. This was a fix suggested by @sleroux .